verify.go

```diff
@@ -118,6 +120,54 @@ func contains(sli []string, ele string) bool {
 	return false
 }
 
+// Returns the Claims from the distributed JWT token
+func resolveDistributedClaim(ctx context.Context, verifier *IDTokenVerifier, src claimSource) ([]byte, error) {
+
+	req, err := http.NewRequest("GET", src.Endpoint, nil)
+	if err != nil {
+		return nil, fmt.Errorf("oidc: malformed request: %v", err)
```

Comment:
nit: this function isn't exported, so something else in the package going to prefix these errors with `oidc:`, we can remove this here.